### DISCUSSION TITLE
Getting Started - Command not found : brew

### DISCUSSION THREAD

**Discussion by Kben15** [Homebrew/discussions#5691] [Getting started]
### Output of `brew config`

```shell
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
zsh: command not found: brew
```

### Description of issue

I installed Homebrew on my Mac (Sonoma 14.1) via the Terminal. I got the "Installation successful" quote at the end of the installation but no command seems to work. 
Everytime I get the " zsh: command not found: brew ". 

Could anyone assist me on this please ? 
Cheers

**Comment by gromgit** [comment]
There was a ***Next Steps*** section after "Installation successful", with instructions on how to add Homebrew to your PATH. Did you follow them?

**Comment by Kben15** [comment]
Hi, 

Thanks for the answer. Problem solved.
